src/pages/editor/Editor.tsx

```diff
@@ -1,30 +1,36 @@
 /* eslint-disable react/no-unknown-property */
-import { Canvas, MeshProps, useFrame } from "@react-three/fiber";
-import { useEffect, useRef } from "react";
-import { useSelector } from "react-redux";
+import { Canvas, MeshProps, useFrame } from '@react-three/fiber';
+import { useEffect, useRef } from 'react';
+import { useSelector } from 'react-redux';
 import * as THREE from 'three';
-import { sceneSelector, changeHoveredState, changeActiveState } from "../../redux/slices/scene";
-import { useAppDispatch } from "../../redux";
+import {
+  sceneSelector,
+  changeHoveredState,
+  changeActiveState,
+} from '../../redux/slices/scene';
+import { useAppDispatch } from '../../redux';
 
-function Box(props: MeshProps) {
-  const meshRef = useRef<THREE.Mesh>(null!)
-  const { hovered, active } = useSelector(sceneSelector)
+function Box({ position }: MeshProps) {
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
for this and following ESLint ignore comments, let's leave them like this for now and let me create a [task](https://ar-chitect.atlassian.net/browse/AS-62) in Jira to address that